pythreepio/README.md

```diff
@@ -0,0 +1,48 @@
+
+![Tests](https://img.shields.io/github/workflow/status/OpenMined/threepio/PyThreepio)
+![License](https://img.shields.io/github/license/OpenMined/threepio)
+![OpenCollective](https://img.shields.io/opencollective/all/openmined)
+
+# Threepio Python Client
+
+Threepio is a multi-language library for translating commands between PyTorch, TensorFlow, and TensorFlow.js 
+This section of the code-base is for the [Python Client](https://pypi.org/project/3p0/).
+
+
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install threepio.
+
+```bash
+pip install 3p0
```

Comment:
As much as I fucking love this package name, you will need to change it: https://github.com/OpenMined/.github/issues/11

Reply:
Yup! I'll update the name in a PR down the line